tools/ZeroC.Ice.Slice.Tools/SliceCompilerTask.cs

```diff
@@ -53,12 +53,12 @@ protected virtual Dictionary<string, string> GetOptions()
             ["OutputDir"] = OutputDir.TrimEnd('\\')
         };
 
-        if (IncludeDirectories != null && IncludeDirectories.Length > 0)
+        if (IncludeDirectories.Length > 0)
```

Comment:
This is an unrelated fix. I updated the properties to be non nullable when imported the builders into the ice repository, but the code was still doing null checks, once of this was causing the rebuilds of Slice files.